lib/formatters/ecto/changeset.ex

```diff
@@ -1,8 +1,24 @@
 defmodule DeliriumTremex.Formatters.Ecto.Changeset do
+  def format({key, [%{} | _] = errors} = _error) when is_list(errors) do
+    errors = Enum.filter(errors, fn x -> not (x == %{}) end)
+    human_key = humanize(key)
+
+    [
+      message: "#{human_key} errors",
+      key: key,
+      messages: nil,
+      full_messages: nil,
+      index: nil,
+      suberrors:
+        Enum.map(errors, fn err -> format({key, err}) end) |> Enum.map(&convert_list_to_map/1)
```

Comment:
Does this convert an array of keyword list objects into an array of maps? If yes, I think that the name of the function is confusing because the function actually converts keyword lists (not lists) into maps.

Further, I think it would be simpler to just do:
```
Enum.map(errors, fn err -> format({key, err}) |> Enum.into(%{}) end
```

Reply:
Done the way you suggested.